### PR TITLE
proto: attribut added in StopTimeUpdate

### PR DESCRIPTION
**Pull request by kadhikari**
https://jira.kisio.org/browse/NAVP-1078

### CODE REVIEW

**Comment by kadhikari** [comment]
SKIPPED_FOR_DETOUR and ADDED_FOR_DETOUR will be transformed to DETOUR in the response navitia

File: type.proto [hunk 1]
```diff
@@ -165,6 +165,7 @@ enum StopTimeUpdateStatus {
     ADDED = 1;
     DELETED = 2;
     UNCHANGED = 3;
+    DETOUR = 4;
```
Comment (TeXitoi) [inline type.proto]: I don't understand the semantic of detour on a stop time. A stop time is added or deleted, it can't be detour.

Comment (pbougue) [reply type.proto]: Same as @TeXitoi , I understood we'd have a "correct" status (I mean understandable for integrators, so ADDED or DELETED) and a string mentionning "detour" somewhere else (like in message). And I'm OK that we bend messages content (or blurry string fields like that).

If I understood correctly, here we won't have a reliable part in Navitia's output. By reliable, I mean a part being consistent from one RT provider to another.
Or will navitia output ADDED or DELETED in another field?

Here (again if I understand correctly) there's no hope someone integrating SNCF's API could have it working for another RT (and no hope for a correct integration at all if we keep DETOUR only).

Comment (TeXitoi) [reply type.proto]: What's the difference between ADDED and ADDED_FOR_DETOUR?

Comment (TeXitoi) [reply type.proto]: And between DELETED and SKIPPED?

Comment (kadhikari) [reply type.proto]: I think we should have a real refinement (discussion).

Comment (nonifier) [reply type.proto]: I think there is a misunderstanding. We only have two cases:
 * `skipped_for_detour` (understand deleted for detour, but there is no such thing as `deleted_for_detour` the API)
 * `added_for_detour`

The difference with `added` and `deleted` is the fact that for a detour, you always have a pair of `skipped_for_detour` along with a `added_for_detour`. This expresses a replacement of the `skipped` stop by the `added` one.

File: type.proto [hunk 2]
```diff
@@ -175,6 +175,7 @@ message StopTimeUpdate {
     optional StopPoint stop_point                   = 5;
     optional StopTimeUpdateStatus departure_status  = 6;
     optional StopTimeUpdateStatus arrival_status    = 7;
+    optional bool is_detour                         = 8;
```
Comment (pbougue) [inline type.proto]: minor: should we add a `[default=false]` ?

**Pull request merged**